tracer/test/Datadog.Trace.Tests/Tagging/InstrumentationTagsTests.cs

```diff
@@ -246,7 +246,7 @@ public void SqlV1Tags_PeerService_DbNameTakesPrecedenceOverOutHost()
             tags.PeerServiceSource.Should().Be("db.instance");
         }
 
-        [Fact]
+[Fact]
```

Comment:
```suggestion
        [Fact]
```

I'm sure you would have seen this as soon as you saw the CI results, but just so you can press "Commit suggestion" 😄